stockflux-securities-master/src/components/Alert.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+import "./Alert.css";
+
+const Alert = ({ messages, type }) => {
+  const [messagesArray, setMessagesArray] = useState([]);
+
+  useEffect(() => {
+    typeof messages === "string"
+      ? setMessagesArray([messages])
+      : setMessagesArray(messages);
+  }, []);
+
+  return (
+    <ul>
+      {messagesArray.map((message, index) => {
+        return (
+          <li key={index}>
+            <div className={`securities-message ${type}`}>{message}</div>
```

Comment:
You don't really need this `div` here either. It would suffice to put the message directly in the `li` which can have a class applied to it as well

Reply:
Yep removed the div now